.github/workflows/ccpp.yml

```diff
@@ -266,7 +266,8 @@ jobs:
               exit 0; \
             fi
 
-  build-and-check-gcc-48:
+  # This job name must be kept in sync with .github/bors.toml.
```

Comment:
Should probably remove the bors mention here

Reply:
Yeah, was just waiting for the other PR to land so I could rebase.